src/hotspot/share/opto/vectorIntrinsics.cpp

```diff
@@ -2041,19 +2041,30 @@ static address get_svml_address(int vop, int bits, BasicType bt, char* name_ptr,
       break;
   }
 
+  if (addr == nullptr && Matcher::supports_scalable_vector()) {
```

Comment:
Shouldn't this be in the `default` branch of the switch above? Otherwise, we would be hitting the `Unimplemented();` at https://github.com/openjdk/jdk/pull/21083/files#diff-33d0866101d899687e04303fb2232574f2cb796ce060528a243ebdc9903b01b1R2040?

Reply:
Do you mean when the bits > 512, e.g. 1024?